.hlint-universum.yaml

```diff
@@ -0,0 +1,414 @@
+###########################################################################
+# Universum
+###########################################################################
+
+- warn: { lhs: Data.Text.pack, rhs: Universum.toText }
+- warn: { lhs: Data.Text.unpack, rhs: Universum.toString }
+
+- warn: { lhs: Data.Text.Lazy.pack, rhs: Universum.toLText }
+- warn: { lhs: Data.Text.Lazy.unpack, rhs: Universum.toString }
+
+- warn: { lhs: Data.Text.Lazy.toStrict, rhs: Universum.toText }
+- warn: { lhs: Data.Text.Lazy.fromStrict, rhs: Universum.toLText }
+
+- warn: { lhs: Data.Text.pack (show x), rhs: Universum.show x }
+- warn: { lhs: Data.Text.Lazy.pack (show x), rhs: Universum.show x }
+
+- warn: { lhs: Control.Exception.evaluate, rhs: evaluateWHNF }
+- warn: { lhs: Control.Exception.evaluate (force x), rhs: evaluateNF x }
+- warn: { lhs: Control.Exception.evaluate (x `deepseq` ()), rhs: evaluateNF_ x }
+
+- warn: { lhs: void (evaluateWHNF x), rhs: evaluateWHNF_ x }
+- warn: { lhs: void (evaluateNF x)  , rhs: evaluateNF_ x }
+
+- suggest: { lhs: nub, rhs: Universum.ordNub,
+    note: "'nub' is O(n^2), 'ordNub' is O(n log n)" }
+
+- warn:
+    lhs: sortBy (comparing f)
+    rhs: Universum.sortOn f
+    note: |
+      If the function you are using for 'comparing' is fast
+      (e.g. 'fst'), use 'sortWith' instead of 'sortOn', because 'sortOn'
+      caches applications the function and 'sortWith' doesn't.
+
+- warn: { lhs: sortOn fst, rhs: Universum.sortWith fst,
+    note: "'sortWith' will be faster here because it doesn't do caching" }
+- warn: { lhs: sortOn snd, rhs: Universum.sortWith snd,
+    note: "'sortWith' will be faster here because it doesn't do caching" }
+- warn: { lhs: sortOn (Down . fst), rhs: Universum.sortWith (Down . fst),
+    note: "'sortWith' will be faster here because it doesn't do caching" }
+- warn: { lhs: sortOn (Down . snd), rhs: Universum.sortWith (Down . snd),
+    note: "'sortWith' will be faster here because it doesn't do caching" }
+
+- warn: { lhs: fmap concat (mapM f s), rhs: Universum.concatMapM f s }
+- warn: { lhs: concat <$> mapM f s, rhs: Universum.concatMapM f s }
+
+# Removed for now since we don't want to make people use (some of) our ad-hoc stuff.
```

Comment:
I'd remove it completely (i. e. this whole commented out stuff), that's indeed ad-hoc and not really useful.